### PR TITLE
Add website intro

### PR DESCRIPTION
**Pull request by neteler**
This PR adds a general introduction to what this website is about.

Besides addressing the general audience, this may also improve the SEO ranking (#24).

### CODE REVIEW

**Comment by neteler** [comment]
Looks like this:

https://sr-org-scratch.github.io/

File: scripts/templates/index.tmpl [hunk 1]
```diff
@@ -23,7 +23,13 @@
     </div>
 
     <div id="content">
+        <h2>About SpatialReference.org</h2>
+        SpatialReference.org is a comprehensive online resource dedicated to providing detailed information and reference materials on spatial coordinate systems, projections, and transformations.<br>
+        This website is essential for geospatial professionals, GIS developers, and cartographers, providing an extensive database of spatial reference systems used globally.<br>
+        The website allows users to search, access, and implement various spatial references easily, offering standardized formats like Well-Known Text (WKT) and Proj4 strings. A wide array of coordinate systems, such as WGS84, UTM, and NAD83, SpatialReference.org are supported.<br>
```
Comment (rouault) [inline scripts/templates/index.tmpl:29]:
```suggestion
        The website allows users to search, access, and implement various spatial references easily, offering standardized formats like Well-Known Text (WKT) and Proj4 strings. A wide array of coordinate systems, such as WGS84, UTM, and NAD83 are supported.<br>
```

File: scripts/templates/index.tmpl [hunk 2]
```diff
@@ -23,7 +23,13 @@
     </div>
 
     <div id="content">
+        <h2>About SpatialReference.org</h2>
+        SpatialReference.org is a comprehensive online resource dedicated to providing detailed information and reference materials on spatial coordinate systems, projections, and transformations.<br>
```
Comment (jjimenezshaw) [inline scripts/templates/index.tmpl:27]: We do not provide any information about transformations (maybe some time in the future, but not near). About projections... well, only the information in the projected CRSs themselves.
```suggestion
        SpatialReference.org is a comprehensive online resource dedicated to providing detailed information and reference materials on spatial coordinate reference systems.<br>
```

File: scripts/templates/index.tmpl [hunk 3]
```diff
@@ -23,7 +23,13 @@
     </div>
 
     <div id="content">
+        <h2>What is SpatialReference.org?</h2>
+        SpatialReference.org is a comprehensive online resource dedicated to providing detailed information and reference materials on spatial coordinate reference systems.<br>
+        This website is essential for geospatial professionals, GIS developers, and cartographers, providing an extensive database of spatial reference systems used globally.<br>
+        The website allows users to search, access, and implement various spatial references easily, offering standardized formats like Well-Known Text (WKT) and Proj4 strings. A wide array of coordinate systems, such as WGS84, UTM, and NAD83 are supported.<br>
```
[the remark this answers is not in the document]
Comment (neteler) [reply scripts/templates/index.tmpl:29]: Like this?

[the remark this answers is not in the document]
Comment (jjimenezshaw) [reply scripts/templates/index.tmpl:29]: > Like this?

Yes.

You can also mention `PROJJSON` as a format (I don't know how much @hobu and @rouault want to "advertise" it. It is very useful to parse some needed information from a CRS without needing a WKT parser).

[the remark this answers is not in the document]
Comment (neteler) [reply scripts/templates/index.tmpl:29]: I'd appreciate a text snippet :)

Comment (jjimenezshaw) [inline scripts/templates/index.tmpl:29]:
```suggestion
        The website allows users to search, access, and implement various spatial references easily, offering standardized formats like Well-Known Text (WKT) or PROJJSON. A wide array of coordinate reference systems, such as WGS 84 or NAD83(2011) geographic systems, thousands of projected systems (like UTM zones), as well as vertical systems like NAVD88 are supported.<br>
```

[the remark this answers is not in the document]
Comment (jjimenezshaw) [reply scripts/templates/index.tmpl:29]: Sorry. Added in other comment.

**Comment by neteler** [comment]
Screenshot above updated again, or see https://sr-org-scratch.github.io/

**Comment by jjimenezshaw** [comment]
Thank you @neteler !

**Pull request merged**